app/javascript/components/patient/Patient.js

```diff
@@ -250,152 +408,248 @@ class Patient extends React.Component {
+                            </span>
+                          </Col>
+                        </Row>
+                      </React.Fragment>
+                    )}
                   </Col>
                 </Row>
               </Col>
-              <Col md="2"></Col>
-              <Col id="exposure-case-information" md="11">
+            </Row>
+            <Row className="g-border-bottom-2 pb-4 mb-2 mt-4 mx-1">
+              <Col id="potential-exposure-information" md="12">
                 <Row>
                   <Col>
                     <div className="float-left">
                       <h5>
-                        {!this.props.details.isolation && <b>POTENTIAL EXPOSURE INFORMATION</b>}
-                        {this.props.details.isolation && <b>CASE INFORMATION</b>}
+                        <b>POTENTIAL EXPOSURE INFORMATION</b>
                       </h5>
                     </div>
-                    <div className="float-right">
+                    <div>
                       {this.props.goto && (
                         <Button variant="link" className="pt-0" onClick={() => this.props.goto(5)}>
-                          <h5>Edit</h5>
+                          <h5>(Edit)</h5>
                         </Button>
                       )}
                     </div>
                     <div className="clearfix"></div>
                   </Col>
                 </Row>
+                <Row className="pt-2">
+                  <Col>
+                    <Row>
+                      <Col>
+                        <b>Last Date of Exposure:</b> <span>{moment(this.props.details.last_date_of_exposure, 'YYYY-MM-DD').format('MM/DD/YYYY')}</span>
+                      </Col>
+                    </Row>
+                    <Row className="pt-2">
+                      <Col>
+                        <b>Exposure Location:</b>{' '}
+                        <span>{this.props.details.potential_exposure_location ? this.props.details.potential_exposure_location : '--'}</span>
+                      </Col>
+                      <Col>
+                        <b>Exposure Country:</b>{' '}
+                        <span>{this.props.details.potential_exposure_country ? this.props.details.potential_exposure_country : '--'}</span>
+                      </Col>
+                    </Row>
+                    <Row className="pt-3">
+                      <Col>
+                        <b>Risk Factors</b>
+                      </Col>
+                    </Row>
+                    {!this.state.hasRiskFactors && <span className="none-text">None</span>}
+                    {this.state.hasRiskFactors && (
+                      <React.Fragment>
+                        {this.props.details.contact_of_known_case && (
+                          <Row>
+                            <Col>
+                              <b className="text-danger">CLOSE CONTACT WITH A KNOWN CASE:</b> <span>{this.props.details.contact_of_known_case_id}</span>
```

Comment:
I think the `:` character should only show if the `id` is present. Same thing for the risk factors below

Reply:
pushed this and some additional cleanup